package.json

```diff
@@ -28,8 +28,9 @@
 		"build-client": "gulp",
 		"build-extensions": "webpack --config ./webpack.config.extensions.js",
 		"build-php": "composer install",
+		"build-production-php": "COMPOSER_MIRROR_PATH_REPOS=1 composer install --no-dev --classmap-authoritative",
```

Comment:
For some reason `--no-dev` is causing our custom `autoload_packages.php` file to not get created. Maybe just need to move autoloader package to `require`?

Reply:
if we added to `require`, then we would also need to `.svnignore` it 

adding to `require` does work fwiw.